repository/src/format.rs

```diff
@@ -16,6 +16,8 @@ pub fn to_semantic_commit(
                 title,
                 body,
                 diff: Diff::None,
+                author: agenda.author.clone(),
+                timestamp: agenda.timestamp,
```

Comment:
If you are going to take the comment above, fix here as well please.

Reply:
reverted